client/src/main/java/uk/co/blackpepper/bowman/Client.java

```diff
@@ -25,44 +25,45 @@
 
 import uk.co.blackpepper.bowman.annotation.RemoteResource;
 
+import static uk.co.blackpepper.bowman.ReflectionSupport.getId;
 import static uk.co.blackpepper.bowman.ReflectionSupport.setId;
 
 /**
  * Class for retrieving, persisting and deleting annotated entity instances via remote
  * hal+json resources.
- *
+ * <p>
  * Entities can contain simple (directly mappable to JSON) properties, and inline or
  * linked associations to further objects.
- * 
+ * <p>
  * <code>Client</code>s are created via {@link ClientFactory#create}.
  *
- * @param <T> the entity type for this client
```

Comment:
If possible could you please revert all the lines that only contain changes to whitespace? Please don't use IDEA's 'Reformat Code' function as the code wasn't originally written in IDEA.

Sorry about this - I know it's not helpful that KEEP_INDENTS_ON_EMPTY_LINES is set to true in our IDEA code style file. I will look at this issue separately.

Reply:
Sure, I can do that. General question: Do you prefer a rebase of the current commits or additional commits to this branch?

Reply:
Please fix your commits rather than adding new ones - you can just force push to the branch in your fork to update this PR.